### PR TITLE
Add plugin installation instructions.

### PR DESCRIPTION
**Pull request by ADmad**

### CODE REVIEW

File: docs/Quick-start-and-introduction.md [hunk 1]
```diff
@@ -1,9 +1,21 @@
 # Quick Start
 
-Add the authentication service to the middleware. See the CakePHP [documentation](http://book.cakephp.org/3.0/en/controllers/middleware.html#) on how to use a middleware if you don't know what it is or how to work with it.
+## Installation
+
+Install the plugin with [composer](https://getcomposer.org/) from your CakePHP Project's ROOT directory (where the **composer.json** file is located)
```
Comment (burzum) [inline docs/Quick-start-and-introduction.md]: `Project's` do you really write that with the`'`?

File: docs/Quick-start-and-introduction.md [hunk 2]
```diff
@@ -1,9 +1,21 @@
...
+```sh
+php composer.phar require cakephp/authentication
+```
+
+Load the plugin by adding following statement in your project's `config/bootstrap.php`
```
Comment (burzum) [inline docs/Quick-start-and-introduction.md]: Missing `the` between "adding" and "following"?

File: docs/Quick-start-and-introduction.md (hunk 1 above)
Comment (dereuromark) [inline docs/Quick-start-and-introduction.md]: Yes.

Comment (markstory) [inline docs/Quick-start-and-introduction.md]: The root directory belongs to the 'project' so the possessive form is correct.

File: docs/Quick-start-and-introduction.md [hunk 3]
```diff
@@ -1,9 +1,21 @@
...
+```php
+Plugin::load('Authentication');
+```
 
 ## Configuration
 
+Add the authentication service to the middleware. See the CakePHP [documentation](http://book.cakephp.org/3.0/en/controllers/middleware.html#) on how to use a middleware if you don't know what it is or how to work with it.
```
Comment (markstory) [inline docs/Quick-start-and-introduction.md]: You don't need 'a' in '... on how to use *a* middleware ...'

Comment (ADmad) [inline docs/Quick-start-and-introduction.md]: I had just moved this existing sentence :)

**Comment by ADmad** [comment]
Updated as per feedback.

**Pull request merged**